packages/grafana-ui/src/components/Logs/getLogRowStyles.ts

```diff
@@ -59,10 +59,6 @@ export const getLogRowStyles = stylesFactory((theme: GrafanaTheme, logLevel?: Lo
       font-size: ${theme.typography.size.sm};
       width: 100%;
     `,
-    logsRowsHorizontalScroll: css`
-      label: logs-rows__horizontal-scroll;
-      overflow: scroll;
-    `,
```

Comment:
@ivanahuckova as `wrap lines` option doesn't affect log rows that are being rendered outside of context provider (at least I didn't notice any difference), I removed that piece of code. And you know what? `LogRowContextProvider` renders properly now! 😎🚀

Reply:
Basically, it was considering that overlay to be within a scrollable space, but when user got a very short list or try to inspect context of edge log lines, it wasn't. Should work properly now.

Reply:
This unfortunately breaks the log-not-wrapped option:
![image](https://user-images.githubusercontent.com/30407135/81271477-6e42b080-904c-11ea-931a-44d8ea656970.png)

Reply:
Weird, on my end it wraps :eyes: Need to investigate it further.